src/Controls/src/Core/Compatibility/Handlers/FlyoutPage/iOS/PhoneFlyoutPageRenderer.cs

```diff
@@ -694,7 +694,9 @@ bool shouldReceive(UIGestureRecognizer g, UITouch t)
 
 							if (_applyShadow)
 							{
-								var openProgress = targetFrame.X / flyoutWidth;
+								var openProgress = !IsRTL
```

Comment:
do we need to apply this logic here also?

https://github.com/dotnet/maui/blob/7cc03f63323979af7282572e480c3ca91e1087fb/src/Controls/src/Core/Compatibility/Handlers/FlyoutPage/iOS/PhoneFlyoutPageRenderer.cs#L666-L669

If yes, then maybe we modify ApplyDetailShadow and pass in all these things as parameters so we're not duplicated.

Reply:
Yeah I've added another commit with a small change that fixes it